README.md

```diff
@@ -131,9 +131,9 @@ pip install --extra-index-url https://pypi.nvidia.com nemo-curator[cuda12x]
     pip install --extra-index-url https://pypi.nvidia.com ".[cuda12x]"
     ```
 
-#### Using Nightly Dependencies for Rapids
+#### Using Nightly Dependencies for RAPIDS
 
-You can also install NeMo Curator using the Rapids nightly, to do so you can set the environment variable `RAPIDS_NIGHTLY=1`.
+You can also install NeMo Curator using the [RAPIDS Nightly Builds](https://anaconda.org/rapidsai-nightly). To do so, you can set the environment variable `RAPIDS_NIGHTLY=1`.
```

Comment:
I would prefer linking to the pip nightly index for RAPIDS or the rapids website with nightly install instructions here rather than the conda channel.